### ISSUE TITLE
[RDS] Add create database and account/privilege support for rds resource 

### ISSUE THREAD

**Issue by Jambo-sudo** [opentelekomcloud/terraform-provider-opentelekomcloud#2684]
Hello Team,

Currently, the RDS (mysql) resource does not support creating database, account and account privilege. May I ask if there are any plans to provide this feature?

**Comment by anton-sidelnikov** [comment]
HI @Jambo-sudo could you attach API documentation for requested features? I couldn't find it.

**Comment by anton-sidelnikov** [comment]
Thank you for reporting the issue! However, in order to help you there are a few preconditions to be met first:
1. Service or feature must be released on PROD environment.
2. Documentation for service or feature must be released on public HelpCenter portal.
3. Functional testing must be completed by QA engineer in the service squad.
4. Regression testing must exist and reports must be available.
5. Link to public HelpCenter portal must be attached in issue.

Please see https://docs-int.otc-service.com/internal-documentation/terraform/process_overview.html#prerequistes-for-new-demands for a more detailed version of this process.

**Comment by TaoLiu-TSI** [comment]
@anton-sidelnikov Hi Anton, @Jambo-sudo is customer, he/she cannot visit the OTC internal documents.

**Comment by TaoLiu-TSI** [comment]
@Jambo-sudo Hi, could you please provide more information about your required features. e.g. which APIs in https://docs.otc.t-systems.com/relational-database-service/api-ref/
or
if there is no your needed APIs, you can also provides the related pages in https://docs.otc.t-systems.com/relational-database-service/umn/
or
other information which we could use to support you. 

Thanks a lot!

**Comment by TaoLiu-TSI** [comment]
https://docs.otc.t-systems.com/relational-database-service/api-ref/permissions_policies_and_supported_actions/rds_actions.html

![image](https://github.com/user-attachments/assets/68953550-0648-46c4-9f9c-718ef833a1b3)


@anton-sidelnikov @Jambo-sudo Hi, Both, Is this what you need?

**Comment by anton-sidelnikov** [comment]
This APIs not tested and not documented

**Comment by Jambo-sudo** [comment]
Hi teams:
We compared Open Telekom Cloud with the Orange Cloud we have used, and noticed some differences in the following RDS (Relational Database Service):

**Comment by Jambo-sudo** [comment]
Hi  Team,
We compared Open Telekom Cloud with the Orange Cloud we have used, and noticed some differences in the RDS (Relational Database Service)：

- Database creation [flexibleengine_rds_database | Resources | FlexibleEngineCloud/flexibleengine | Terraform | Terraform Registry](https://registry.terraform.io/providers/FlexibleEngineCloud/flexibleengine/latest/docs/resources/rds_database)

- User creation and management [flexibleengine_rds_account | Resources | FlexibleEngineCloud/flexibleengine | Terraform | Terraform Registry](https://registry.terraform.io/providers/FlexibleEngineCloud/flexibleengine/latest/docs/resources/rds_account)

- User permission assignment [flexibleengine_rds_database_privilege | Resources | FlexibleEngineCloud/flexibleengine | Terraform | Terraform Registry](https://registry.terraform.io/providers/FlexibleEngineCloud/flexibleengine/latest/docs/resources/rds_database_privilege)
	 
In Orange Cloud, these features are well-supported, which greatly facilitates our infrastructure management. However, the current Open Telekom Cloud Terraform provider lacks these essential functionalities, limiting our ability to fully automate and manage cloud resources during migration process. 
We believe that directly providing an API for us to implement these features is not the most suitable solution. Instead, we suggest that the Open Telekom Cloud Terraform team collaborate with the RDS team to address this gap. We recommend referring to Orange Cloud's Terraform provider as a model to implement similar capabilities for Open Telekom Cloud.
This enhancement would greatly improve the usability of your Terraform provider, enabling more efficient infrastructure management for users like us who rely heavily on Terraform for automation.

Thank you for your attention to this matter. We look forward to seeing improvements in this area ASAP.
